packages/card/MediaCard.jsx

```diff
@@ -111,7 +115,7 @@ export default function MediaCard({
         )}
         {type && (
           <div className="su-text-18 md:su-text-16 su-mb-15 md:su-mb-19 su-gap-6 su-text-black-70 dark:su-text-black-50 su-flex su-nowrap su-items-center su-leading-none">
-            <FAIcon icon={typeIcon} set="solid" />
+            <FAIcon icon={typeIcon} set="solid" data-testid={iconTestId} />
```

Comment:
Is the work in this file adding the magazine icon option to the Single Feature Media component ?

Reply:
If so, do you have a dev page I am supposed to test this piece out on as well?

Reply:
No. This is just an update for the tests that run through MediaCard.test.jsx. When I did the work to add the magazine I didn't realize a test was failing because the data attribute looking for the icon was removed. To test yourself, pull the branch down and run `npm run test`. You should see that all tests pass.

Note: When running `npm run test` you will see a React error related to the `popular-stories` component. This has been in the code for some time and was introduced by Squiz. We'll have to look closer to resolve but this is not related to the work on this ticket or the magazine ticket.